src/reader/_parser.py

```diff
@@ -708,7 +709,16 @@ def _process_feedparser_dict(url: str, d: Any) -> FeedAndEntries:
     # This must be a list, not a generator expression,
     # otherwise the user may get a ParseError when calling
     # next(parse_result.entries), i.e. after parse() returned.
-    entries = [_feedparser_entry(url, e, is_rss) for e in d.entries]
+    entries = []
+    for e in d.entries:
+        try:
+            entry = _feedparser_entry(url, e, is_rss)
+            entries.append(entry)
+        except ParseError as exc:
+            warnings.warn(f"{exc.url}: {exc.message}")
```

Comment:
In general, it is better to use library-specific warning classes, so users can filter by the warning class without having to look at the message.

I address this in https://github.com/lemon24/reader/commit/57f504ebdf6c8c96011f572c8ffdd6d9d17d17b0. I chose to make ParseError a ReaderWarning too, so that users can opt into them being raised with something like `warnings.simplefilter('error', category=ParseError)`.

Reply:
Thanks for the tip!

Honestly, I don't have much experience with warnings. I think I tried emitting warning of `ParseError` category (having read that all warnings ultimately inherit from `Exception`), but that didn't work right away. I decided to submit PR that fixes core of the issue, and maybe polish it later based on your comments.

Reply:
No worrie!  I first tried using ParseError directly as well; by the time I figured out the right way of doing it, the code was already changed :)

Thank you again for submitting this!

(BTW, I'll try to cut a PyPI release by the end of the week.)